packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -167,6 +205,26 @@ export function BlockSettingsDropdown( {
 								<__unstableBlockSettingsMenuFirstItem.Slot
 									fillProps={ { onClose } }
 								/>
+								{ firstParentClientId !== undefined && (
+									<MenuItem
+										{ ...showParentOutlineGestures }
+										ref={ selectParentButtonRef }
+										icon={
+											<BlockIcon
+												icon={ parentBlockType.icon }
+											/>
+										}
+										onClick={ () =>
+											selectBlock( firstParentClientId )
+										}
+									>
+										{ sprintf(
+											/* translators: %s: Name of the block's parent. */
+											__( 'Select parent (%s)' ),
```

Comment:
```suggestion
											__( 'Select parent block (%s)' ),
```

Reply:
I would figure that the "block" could be inferred here from the context of this being the block toolbar's ellipsis menu, but I've committed the change anyway. We can always shorten it later if it feels too long.